legi/tar2sqlite.py

```diff
@@ -476,18 +478,30 @@ def main():
     if not os.path.isdir(args.anomalies_dir):
         os.mkdir(args.anomalies_dir)
 
-    db = connect_db(args.db)
-    for pragma in args.pragma:
-        query = "PRAGMA " + pragma
-        result = db.one(query)
-        print("> Sent `%s` to SQLite, got `%s` as result" % (query, result))
+    db = connect_db(args.db, pragmas=args.pragma)
+    last_update = db.one("SELECT value FROM db_meta WHERE key = 'last_update'")
 
-    process_links = not args.skip_links
-    if args.skip_links:
+    # Check and record the data mode
+    db_meta_raw = db.one("SELECT value FROM db_meta WHERE key = 'raw'")
+    if args.raw:
+        versions_brutes = db.one("SELECT count(*) FROM textes_versions_brutes")
+        data_is_not_raw = versions_brutes > 0 or db_meta_raw is False
+        if data_is_not_raw:
```

Comment:
je bug sur la double négation :)

```python
data_is_raw = versions_brutes == 0 or db_meta_raw is True
if not data_is_raw:
    print("!> Can't honor --raw option, the data has already been modified previously.")
    raise SystemExit(1)
```

Reply:
Dans le cas présent `db_meta_raw is True` n'est pas l'opposé de `db_meta_raw is False`, car il y a aussi `None` à prendre en compte. Du coup il faudrait utiliser `db_meta_raw is not False`, mais ça ne ferait que remplacer une forme de double négation par une autre.